src/services/db/TokenService.ts

```diff
@@ -0,0 +1,340 @@
+import { AxiosResponse } from "axios"
+import {
+  Result,
+  ok,
+  err,
+  ResultAsync,
+  okAsync,
+  errAsync,
+  fromPromise,
+} from "neverthrow"
+import { ModelStatic } from "sequelize"
+
+import logger from "@logger/logger"
+
+import DatabaseError from "@errors/DatabaseError"
+import NoAvailableTokenError from "@errors/NoAvailableTokenError"
+import TokenParsingError from "@errors/TokenParsingError"
+
+import { AccessToken } from "@database/models"
+
+// Env vars
```

Comment:
I think this comment is misplaced right?

Reply:
Yes, will remove this comment.